frontend/src/pages/MeetingCreatePage/index.tsx

```diff
@@ -59,7 +33,17 @@ const MeetingCreatePage = () => {
   });
   const currentDate = new Date();
   const isParticipantSelected = selectedItems.length > 0;
-  const userState = useContext(userContext);
+
+  const meetingCreateMutation = useMutation(createMeetingApi, {
+    onSuccess: ({ body: { id } }) => {
+      alert('모임 생성을 완료했습니다.');
+      navigate(`/meeting/${id}`);
+    },
+    onError: (error) => {
+      alert(error);
+      alert('모임 생성을 실패했습니다.');
+    },
```

Comment:
alert가 두개있네요? 아래만 남겨도 될 것 같습니다.

```suggestion
    onError: () => {
      alert('모임 생성을 실패했습니다.');
    },
```

Reply:
앗 삭제했습니다.